lisa/tools/nfs.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+from typing import List
+
+from assertpy.assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import SLES, Debian, Redhat
+from lisa.tools import Mount
+from lisa.util import LisaException
+
+
+class NFS(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def can_install(self) -> bool:
+        return True
+
+    def setup_server(self, client_ips: List[str], dir_name: str) -> None:
+        # run installation
+        self._install()
+
+        # create directory to share
+        self.node.execute(f"chown -R nobody:nogroup {dir_name}", sudo=True)
+
+        # add client ip to /etc/exports file
+        for client_ip in client_ips:
+            self.node.execute(
+                f"echo '{dir_name} {client_ip}(rw,sync,no_subtree_check)' "
+                ">> /etc/exports",
+                shell=True,
+                sudo=True,
+            )
+
+        # restart nfs service
+        if isinstance(self.node.os, Redhat):
+            self._disable_firewall()
+            self.node.execute(
+                "service nfs restart",
+                sudo=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message="nfs service failed to start",
+            )
+        elif isinstance(self.node.os, Debian):
+            self.node.execute(
+                "service nfs-kernel-server restart",
+                sudo=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message="nfs service failed to start",
+            )
+        elif isinstance(self.node.os, SLES):
+            self.node.execute(
+                "service nfsserver restart",
+                sudo=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message="nfs service failed to start",
+            )
+        else:
+            raise LisaException(f"{self.node.os} not supported")
+
+    def setup_client(
+        self,
+        server_ip: str,
+        server_shared_dir: str,
+        mount_dir: str,
+        protocol: str = "tcp",
+    ) -> None:
+        # run installation
+        self._install()
+
+        # mount server shared directory
+        self.node.shell.mkdir(PurePosixPath(mount_dir), exist_ok=True)
+        self.node.tools[Mount].run(
+            f"-t nfs -o proto={protocol},vers=3  "
```

Comment:
check if `Mount.mount` can be used. If not, add arguments to make it useful. If it needs to add arguments, make sure their names are the same as mount's help content.